docs/src/howto-guides/017_apama_software_management_plugin.md

```diff
@@ -1,70 +1,58 @@
 # Apama Software Management Plugin
 
-The apama plugin can be used to install and manage apama artefacts.
-This plugin can be used to install an apama project on a device as well as inject monitor files into an already installed project.
+The Apama plugin, part of the Apama thin-edge support package, can be used to install Apama projects using the Cumulocity software management feature.
 
+## Add the Apama thin-edge support package to the tenant software repository and deploy it to thin-edge
```

Comment:
```suggestion
## Prerequisites

The Apama thin-edge support package must be installed before you can install Apama projects on thin-edge.
If not already installed, it can also be installed from the Cumulocity software repository as follows:
```